api4/bot.go

```diff
@@ -100,9 +101,9 @@ func getBot(c *Context, w http.ResponseWriter, r *http.Request) {
 	}
 	botUserId := c.Params.BotUserId
 
-	includeDeleted := false
-	if iD, err := strconv.ParseBool(r.URL.Query().Get("include_deleted")); err == nil {
-		includeDeleted = iD
+	includeDeleted, e := strconv.ParseBool(r.URL.Query().Get("include_deleted"))
```

Comment:
Apologies for our use of `*model.AppError` that forces a new name to be picked here. Traditionally, we rename the /other/ errors to be `appErr`, and leave the stdlib ones as `err`. Could you help rename as such below, e.g. `bot, appErr := ...`, and leave this one as the more traditional `err`?

Reply:
Sure thing! :) Thank you!